src/main/java/cubicchunks/asm/mixin/core/client/MixinWorldClient.java

```diff
@@ -47,15 +47,12 @@
 	@Shadow public abstract boolean invalidateRegionAndSetBlock(BlockPos pos, IBlockState blockState);
 
 	@Override public void initCubicWorld() {
+		super.initCubicWorld();
```

Comment:
In my tests it worked just fine, I tweaked the config and everything appeared to work normally. Is there any reason it wouldn't work?